Source/Microsoft.Teams.Apps.FAQPlusPlus.Configuration/Controllers/HomeController.cs

```diff
@@ -0,0 +1,290 @@
+﻿// <copyright file="HomeController.cs" company="Microsoft">
+// Copyright (c) Microsoft. All rights reserved.
+// </copyright>
+
+namespace Microsoft.Teams.Apps.FAQPlusPlus.Configuration.Controllers
+{
+    using System;
+    using System.Net;
+    using System.Threading.Tasks;
+    using System.Web;
+    using System.Web.Mvc;
+    using Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker;
+    using Microsoft.Teams.Apps.FAQPlusPlus.Common.Models;
+    using Microsoft.Teams.Apps.FAQPlusPlus.Common.Providers;
+
+    /// <summary>
+    /// Home Controller
+    /// </summary>
+    [Authorize]
+    public class HomeController : Controller
+    {
+        private const string TeamIdEscapedStartString = "19%3a";
+        private const string TeamIdEscapedEndString = "%40thread.skype";
```

Comment:
As we discussed, we need to fix this else it will not work for CSv2 threads.

Reply:
will create a new PR for this